lib/api.js

```diff
@@ -93,7 +92,7 @@ api.declare({
   route:       '/secrets/:name(*)',
   deferAuth:   true,
   name:        'remove',
-  scopes:      [['secrets:remove:<name>']],
+  scopes:      [['secrets:set:<name>']],
```

Comment:
Hm, there may actually be a use-case for that in the signing world, where you would want a task to be able to fetch a secret and then delete it.  But let's wait until we have that need -- as you say, easy enough to do.